src/modules/fields/nestedFields.js

```diff
@@ -25,7 +24,6 @@ export const nestedFields = {
     columns: AXIS,
     rows: AXIS,
     filters: AXIS,
-    user: USER,
```

Comment:
You intended to remove the whole `user` object, right? Not just `userCredentials`

Reply:
I think since `user` is always returned with a set list of fields as you also mentioned above, it can be removed here where before we were only fetching `name` and `userCredentials`.
The change is then from `user[name,userCredentials[username]]` to `user`.